packages/dm-core-plugins/src/attribute-selector/AttributeSelectorPlugin.tsx

```diff
@@ -46,7 +48,12 @@ export const AttributeSelectorPlugin = (props: IUIPlugin): JSX.Element => {
     if (config.childTabsOnRender) {
       const newChildTabs: TStringMap = {}
       Object.entries(entity).forEach(([key, attributeData]: [string, any]) => {
-        if (typeof attributeData == 'object') {
+        if (
+          (config?.visibleAttributes === undefined ||
+            config?.visibleAttributes.length === 0 ||
+            config?.visibleAttributes.includes(key)) &&
+          typeof attributeData == 'object'
+        ) {
```

Comment:
Perhaps more readable?
```suggestion
        const filteredOutInConfig = 
          config?.visibleAttributes !== undefined &&
          config?.visibleAttributes.length > 0 &&
          !config?.visibleAttributes.includes(key)
        if !filteredOutInConfig && typeof attributeData == 'object') {
```

Reply:
fixed